frontend/src/components/loading-buffer/loading-buffer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { CircularProgress } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+export const Loading = ({ loading }) => {
```

Comment:
Not sure why you pass the state with it, you can just call the component when it's "loading" and then disable it again

Reply:
I will fix it later when more components call the backend